core/src/main/java/com/google/common/truth/GuavaOptionalSubject.java

```diff
@@ -21,6 +21,9 @@
 /**
  * Propositions for Guava {@link Optional} subjects.
  *
+ * <p>If you are looking a {@code java.util.Optional} subject, please read
```

Comment:
@kluever Rather than "looking a", I wonder if this should say "looking for a". 😉

Reply:
Durp, thanks!